bin/boot

```diff
@@ -54,10 +84,11 @@ then
 else
     file=$ODOO_CONF
 fi
-if [ ! -f "$file" ]; then
-    cp /opt/sources/odoo.conf $file 
-    echo "Copy template as conf file $file"
+if [ -f "$file" ]; then
```

Comment:
@seb-elico current the logic is just remove the one odoo.conf inside container , then copy new one from the image and build result to $file path. It wil trigger every time when the container up.